src/components/Calculator.js

```diff
@@ -1,11 +1,84 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
-import Functionality from './Functionality';
+import calculate from '../logic/calculate';
 
-const Calculator = () => (
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/226892692-3f0e8978-ec28-40c0-be73-9898b3c8108b.png)

**[OPTIONAL]**

- [ ] It would be great if you remove this Live Demo section to make your readme enough descriptive and professional :+1: because now you don't have any live link for your application so you can remove this section :blush: Once you would have a live link then you can add this Live Demo section again with a working live link :+1: